packit/distgit.py

```diff
@@ -179,7 +179,12 @@ def upload_to_lookaside_cache(self, archive_path: str) -> None:
         logger.info("uploading to the lookaside cache")
         f = FedPKG(self.fas_user, self.local_project.working_dir)
         f.init_ticket()
-        f.new_sources(sources=archive_path)
+        try:
+            f.new_sources(sources=archive_path)
+        except Exception as ex:
+            logger.error("Your Fedora kerberos ticket is NOT valid: %r", ex)
```

Comment:
There could be also other problems: the user does not have perms, there is a network outage, ...

The error message should reflectt that: what went wrong. You can also provide info on common symptoms.

Reply:
I have updated it a message a bit.